hazelcast/src/main/java/com/hazelcast/sql/SqlService.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.sql;
+
+/**
+ * Service to execute SQL queries.
+ */
+public interface SqlService {
+    /** Unique service name. */
+    String SERVICE_NAME = "hz:impl:sqlService";
```

Comment:
I think these are not meant to be public - seems we're conflating the public "SQL service" with the internal "Hazelcast SQL Service Implementation"

Reply:
Removed. This service name is only required by a client implementation (which is not ready yet). It will be located in a private class.